doc/nrf/protocols/bt_mesh/configuring.rst

```diff
@@ -108,7 +109,7 @@ The following configuration options are used to configure the Bluetooth Low Ener
 
 * :kconfig:option:`CONFIG_BT_COMPANY_ID` - Sets the Bluetooth Company Identifier for this device.
 * :kconfig:option:`CONFIG_BT_DEVICE_NAME` - Defines the Bluetooth device name.
-* :kconfig:option:`CONFIG_BT_L2CAP_TX_MTU` - Sets the maximum L2CAP MTU for L2CAP TX buffers.
+* :kconfig:option:`CONFIG_BT_L2CAP_TX_MTU` - Sets the maximum L2CAP MTU for L2CAP TX buffers. When GATT is enabled, recommended value is :kconfig:option:`CONFIG_BT_BUF_ACL_TX_SIZE`-4.
```

Comment:
Why `CONFIG_BT_BUF_ACL_TX_SIZE-4` is recommended value?

Reply:
`CONFIG_BT_BUF_ACL_TX_SIZE` is `CONFIG_BT_L2CAP_TX_MTU` + L2CAP Header which is 4 bytes long.